pgmpy/models/MarkovModel.py

```diff
@@ -190,14 +195,22 @@ def get_cardinality(self):
         >>> factor = DiscreteFactor(['Alice', 'Bob'], cardinality=[2, 2],
         ...                 values=np.random.rand(4))
         >>> student.add_factors(factor)
+        >>> student.get_cardinality('Alice')
```

Comment:
@lohani2280 It's generally better to use named arguments to keep things clear. So, `student.get_cardinality(node='Alice')` would be better in my opinion.